connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -131,7 +133,7 @@ public void onCompletion(Throwable error, ConsumerRecord<String, String> record)
     @Before
     public void setUp() throws Exception {
         store = PowerMock.createPartialMock(KafkaBasedLog.class, new String[]{"createConsumer", "createProducer"},
-                TOPIC, PRODUCER_PROPS, CONSUMER_PROPS, consumedCallback, time);
+                TOPIC, PRODUCER_PROPS, CONSUMER_PROPS, consumedCallback, time, TOPIC_SUPPLIER);
```

Comment:
Probably one test, perhaps just with a mock, that validates the `TOPIC_SUPPLIER` is invoked?